.github/workflows/cont_integration.yml

```
@@ -126,16 +117,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
-      - uses: actions-rs/toolchain@v1
+      - uses: dtolnay/rust-toolchain@stable
         with:
             # we pin clippy instead of using "stable" so that our CI doesn't break
             # at each new cargo release
             toolchain: "1.67.0"
```

```suggestion
      # we pin clippy instead of using "stable" so that our CI doesn't break
      # at each new cargo release
      - uses: dtolnay/rust-toolchain@1.67.0
```

This could be simplified to this if desired